xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingMarketDataService.java

```diff
@@ -183,28 +192,32 @@ public void openSubscriptions(ProductSubscription productSubscription) {
   }
 
   private final class OrderbookSubscription {
-    long snapshotlastUpdateId;
-    AtomicLong lastUpdateId = new AtomicLong(0L);
+    final Observable<DepthBinanceWebSocketTransaction> stream;
```

Comment:
why was this pulled out of the `BinanceWebsocketTransaction`

Reply:
That's just small optimisation, to avoid mapping an object twice. As that observable is shared for both subscriptions of order book updates and order book.